src/scenes/AcceleratorRouter/Overview/index.tsx

```diff
@@ -25,7 +76,31 @@ const OverviewView = () => {
           <Button id='new-cohort' label={strings.ADD_COHORT} icon='plus' onClick={goToNewCohort} size='medium' />
```

Comment:
This button should be removed, the CTA will be within each tabbed view.

Reply:
I intentionally left this in-place for now, it can be removed when the new tabbed views are added.